test/ProjectService.Data.Provider.UnitTests/ProjectService.Data.Provider.MsSql.Ef.UnitTests.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
We do not need to test this layer. Remove this project

Reply:
Ok, I am not still this removed. I will be update this now.

Reply:
Fixed